ue4docker/infrastructure/BuildConfiguration.py

```diff
@@ -27,6 +27,23 @@
 # The default memory limit (in GB) under Windows
 DEFAULT_MEMORY_LIMIT = 10.0
 
+class VisualStudio(object):
+	VS2017 = '2017'
+	VS2019 = '2019'
+
+	BuildNumbers = {
+		'2017' : '15',
+		'2019' : '16',
+	}
+
+	MinSupportedUnreal = {
+		# Unreal Engine 4.23.1 is the first that successfully builds with Visual Studio v16.3
+		# See https://github.com/EpicGames/UnrealEngine/commit/2510d4fd07a35ba5bff6ac2c7becaa6e8b7f11fa
+		#
+		# Unreal Engine 4.25 is the first that works with .NET SDK 4.7+
```

Comment:
With the current change, we still have the 4.6.2 SDK installed under VS2019, so I'd hope we can use VS 2019 against 4.23.1 now, if the user desires.

Reply:
We can't (read other thread in this PR). We have 4.8 SDK under VS2019 Build Tools and older versions of UE4 are unable to use it.

I should possibly put a comment somewhere in the code about this situation.